examples/eks/addon.yaml

```diff
@@ -0,0 +1,16 @@
+---
+apiVersion: eks.aws.crossplane.io/v1alpha1
+kind: Addon
+metadata:
+  name: sample-addon
+  labels:
+    example: "true"
+spec:
+  forProvider:
+    region: eu-central-1
+    addon: coredns
+    addonVersion: "1.8.4"
```

Comment:
change Version to     `addonVersion: "v1.8.4-eksbuild.1"`

Reply:
Fixed